frontend/src/metabase/components/EntityItem.jsx

```diff
@@ -138,7 +138,7 @@ function EntityItemMenu({
   }
   return (
     <EntityMenuContainer align="center">
-      {!isPinned && (
+      {showPinnedAction && (
```

Comment:
I now can't see the pin button with write-permissions to the collection. `showPinnedAction = onPin && isPinned`, so it means it should be shown for pinned items which is not the case.

Reply:
oops, misread that, added a second boolean, `showUnpinnedAction`